apps/discord-bot/src/commands/murdermystery/murdermystery.profile.tsx

```diff
@@ -8,76 +8,232 @@
 
 import { BaseProfileProps } from "../base.hypixel-command";
 import { Container, Footer, Header, SidebarItem, Table } from "#components";
-import { FormattedGame, MurderMysteryMode } from "@statsify/schemas";
-import { LocalizeFunction } from "@statsify/discord";
+import { FormattedGame, GameMode, MurderMysteryModes } from "@statsify/schemas";
+import { formatTime } from "@statsify/util";
 
-interface MurderMysteryModeColumnProps {
-  title: string;
-  stats: MurderMysteryMode;
-  t: LocalizeFunction;
+export interface MurderMysteryProfileProps extends BaseProfileProps {
+  mode: GameMode<MurderMysteryModes>;
 }
 
-const MurderMysteryModeColumn = ({ title, stats, t }: MurderMysteryModeColumnProps) => (
-  <Table.ts title={`§6${title}`}>
-    <Table.td title={t("stats.gamesPlayed")} value={t(stats.gamesPlayed)} color="§e" />
-    <Table.td title={t("stats.wins")} value={t(stats.wins)} color="§a" />
-    <Table.td title={t("stats.kills")} value={t(stats.kills)} color="§c" />
-    <Table.td title={t("stats.kdr")} value={t(stats.kdr)} color="§b" />
-  </Table.ts>
-);
-
 export const MurderMysteryProfile = ({
   player,
   background,
   logo,
   skin,
   t,
   badge,
+  mode,
   user,
   time,
-}: BaseProfileProps) => {
+}: MurderMysteryProfileProps) => {
   const { murdermystery } = player.stats;
 
   const sidebar: SidebarItem[] = [
     [t("stats.coins"), t(murdermystery.coins), "§6"],
     [t("stats.lootChests"), t(murdermystery.lootChests), "§e"],
-    [t("stats.murdererWins"), t(murdermystery.murdererWins), "§c"],
-    [t("stats.detectiveWins"), t(murdermystery.detectiveWins), "§b"],
-    [t("stats.heroWins"), t(murdermystery.heroWins), "§a"],
+    [t("stats.goldPickedUp"), t(murdermystery[mode.api].goldPickedUp), "§6"],
+    [t("stats.overallWins"), t(murdermystery[mode.api].wins), "§2"],
```

Comment:
This should really just go with the other wins in the table. Having the separate is confusing plus I think overall wins is probably more important than the individual ones so it should be big like the other types of wins